cadquery/occ_impl/exporters/assembly.py

```diff
@@ -23,34 +25,62 @@
 from OCP.Message import Message_ProgressRange
 from OCP.Interface import Interface_Static
 
-from ..assembly import AssemblyProtocol, toCAF, toVTK
+from ..assembly import AssemblyProtocol, toCAF, toVTK, toFusedCAF
 from ..geom import Location
 
 
-def exportAssembly(assy: AssemblyProtocol, path: str, **kwargs) -> bool:
+class ExportModes:
+    DEFAULT = "DEFAULT"
+    FUSED = "FUSED"
+
+
+STEPExportModeLiterals = Literal["DEFAULT", "FUSED"]
```

Comment:
To keep things more consistent, I'd propose to use lowercase for options that are not abbreviations.
```suggestion
STEPExportModeLiterals = Literal["default", "fused"]
```